.travis.yml

```diff
@@ -22,6 +22,7 @@ addons:
    - libexpat1-dev
    - libipc-run-perl
    - poppler-utils
+   - nodejs
```

Comment:
Since Travis no longer supports OSS projects, this will need to be a GitHub Action or something else. I'm still happy to support linting against our editorconfig setup, just not in Travis. Since I use `package.json`'s `devDependencies` for commitlint stuff anyway this could also be added there. I bet there is a ready made GH Action too.